src/libmongoc/examples/cmake/vcpkg/app.c

```diff
@@ -0,0 +1,9 @@
+#include <mongoc/mongoc.h>
+
+#include <stdio.h>
+
+int
+main ()
```

Comment:
A subtle difference between C and C++, it is advised by the ISO C99 standard to use the signature `int main(void)` for a `main` function that takes no arguments. See section `5.1.2.2.1 Program startup` in the link below:
```suggestion
main (void)
```

https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf